src/DataFixtures/ReviewFixtures.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\DataFixtures;
+
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use App\Entity\Review;
+
+class ReviewFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $review = new Review();
+        $review->setReviewStars('5');
+
+        $review->setReviewComment('Amazing course');
```

Comment:
It seems like this file stands out, because of different spacing. Till right now it was empty line after new object. It would be cool to keep like that everywhere. :+1:

Reply:
Will do ;)